apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -530,297 +539,605 @@ public Flow(long maxByteCount, long maxElementCount, Action<Task> registerTaskFn
                     {
                         _event.Set();
                     }
-                })));
+                }));
+                _registerTaskFn(task);
+            }
+        }
+
+        // internal for testing.
+        internal class ReQueue<T>
```

Comment:
Ah, this is the class from earlier. Let's think up a different name that doesn't involve the type and method name differing only be the case of a single letter.

Reply:
OK, renamed it "RequeueableQueue" for now. It's internal only so isn't desperately important.
Also, I like the word "Requeueable" because it has five vowels in a row!